src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -327,8 +325,12 @@ internal bool IsRuntimeAsyncEnabledIn(MethodSymbol method)
                 return false;
             }
 
-            // PROTOTYPE: Check for attributes that turn on/off the feature member-by-member
-            return true;
+            return method switch
+            {
+                SourceMethodSymbol { IsRuntimeAsyncEnabledInMethod: ThreeState.True } => true,
+                SourceMethodSymbol { IsRuntimeAsyncEnabledInMethod: ThreeState.False } => false,
+                _ => Feature("runtime-async") == "on"
+            };
```

Comment:
Let's update the compiler spec for how those three flags (runtime feature flag, compiler feature flag, method-level attribute) work together